app/controllers/discovered_hosts_controller.rb

```diff
@@ -184,16 +184,22 @@ def auto_provision_all
 
   private
 
+  def remove_slashes(string)
+    string.slice!(0)
+    string.slice!(string.length-1)
+    string
+  end
```

Comment:
What are you doing here? Setting will work as-is, we don't need to remove first and last character.